File: src/main/java/com/example/ssauc/user/history/dto/ProductReportDto.java
package com.example.ssauc.user.history.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProductReportDto {
    private Long reportId;
    private Long productId;
    private String productName;
    private String productImageUrl;
    private String reportReason;
    private LocalDateTime reportDate;
    private LocalDateTime processedAt;
    private String status;
}

File: src/main/java/com/example/ssauc/user/history/dto/ProductReportDto.kt
package com.example.ssauc.user.history.dto

import lombok.*
import java.time.LocalDateTime

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
class ProductReportDto {
    public val reportId: Long? = null
    public val productId: Long? = null
    public val productName: String? = null
    public val productImageUrl: String? = null
    public val reportReason: String? = null
    public val reportDate: LocalDateTime? = null
    public val processedAt: LocalDateTime? = null
    public val status: String? = null
}
